regulations/static/regulations/css/less/module/comment.less

```diff
@@ -63,6 +59,10 @@ comment.less contains styles related to commenting on sections
   position: relative;
 }
 
+#preamble-read .node {
+  padding-right: 250px;
+}
```

Comment:
Note, however, that a fixed width here means that the text doesn't expand when the user closes the ToC -- everything just gets shifted left.

Would it make sense to use a percentage both here and in `#activate-write`? Perhaps with a minimum?